app/views/asset/show_overview.scala.html

```diff
@@ -250,6 +250,13 @@
           <td>SCSI Storage</td>
           <td>@aa.lshw.totalStorage.toHuman</td>
         </tr>
+	@if(aa.lshw.hasFlashStorage != true) {
```

Comment:
is this logic backwards, or am i crazy? shouldnt it be `@if(aa.lshw.hasFlashStorage)`?

Reply:
Uh, yup. Pushed a commit from testing stuff 🤦‍♂️ One sec, i'll fix it.